deploy/terraform/aws/lambda/variables.tf

```diff
@@ -4,6 +4,16 @@ variable "aws_region" {
   default     = "us-east-1"
 }
 
+variable "aws_access_key" {
```

Comment:
Let's use named profiles (with `profile`) instead of `aws_access_key` and `aws_secret_access_key`. Static creds are increasingly 💀 .

https://registry.terraform.io/providers/hashicorp/aws/latest/docs#profile

https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html#cli-configure-files-using-profiles

Reply:
Fixed it 👌